docs/workers.md

```diff
@@ -210,7 +210,7 @@ expressions:
     ^/_matrix/federation/v1/get_groups_publicised$
     ^/_matrix/key/v2/query
     ^/_matrix/federation/unstable/org.matrix.msc2946/spaces/
-    ^/_matrix/federation/unstable/org.matrix.msc2946/hierarchy/
+    ^/_matrix/federation/(v1|unstable/org.matrix.msc2946)/hierarchy/
```

Comment:
Does this want to also say "v3" now that Matrix 1.1 is specced?

Reply:
No, new endpoints star at `v1` (note that this matches what is in the MSC also).

Reply:
Also federation endpoints have always been versioned per-endpoint so have always started at `v1` (they never had `r0` paths).